buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -19,7 +19,7 @@ object Versions {
     const val okhttp = "4.11.0"
 
     const val firebaseAdmin = "9.1.0"
-    const val radarSchemas = "0.8.4"
+    const val radarSchemas = "0.8.7-SNAPSHOT"
```

Comment:
Should the next version be `0.8.6-SNAPSHOT`?

Reply:
I dont think so, dev branch for schemas is at 8.7